usaspending_api/disaster/tests/fixtures/disaster_account_data.py

```diff
@@ -40,7 +40,7 @@ def disaster_account_data():
         submission_fiscal_year=2022,
         submission_fiscal_quarter=3,
         submission_fiscal_month=8,
-        submission_reveal_date="2022-4-15",
+        submission_reveal_date="2020-4-15",
```

Comment:
The reason this was changed is that the covid-19 sql view checks against the submission reveal date while joining to the dabs window table. The original code apparently did not have the same checks, so either this data has to change or the expected results do.

Reply:
Are you talking about this? `dabs.submission_reveal_date <= now()`?

The data and results _should_ match. the issue could be pytests are mocking the system datetime while the database is not

Reply:
Yeah, the issue is that for the data in question, the submission reveal date is `"2022-4-15"`, which is not `<= now()` so the covid19_faba_view will not include that data, but the test is written to include that data. From my understanding of the business logic here, we don't want to include that data, since the reveal date is in the future.

Reply:
Since the pytests are mocking the now() function in python the optimal path forward would be to mock/reset the database datetime for the tests. Otherwise for consistency many many tests will need to be changed to be in the past instead of the future